smallrye-reactive-messaging-kafka/src/test/java/io/smallrye/reactive/messaging/kafka/CountKafkaCdiEvents.java

```diff
@@ -0,0 +1,89 @@
+package io.smallrye.reactive.messaging.kafka;
+
+import java.lang.annotation.Annotation;
+import java.util.concurrent.CompletionStage;
+import java.util.concurrent.atomic.LongAdder;
+
+import javax.enterprise.event.Event;
+import javax.enterprise.event.NotificationOptions;
+import javax.enterprise.util.TypeLiteral;
+
+import org.apache.kafka.clients.consumer.Consumer;
+import org.apache.kafka.clients.producer.Producer;
+
+public class CountKafkaCdiEvents extends KafkaCDIEvents {
+    public static final KafkaCDIEvents noCdiEvents = new CountKafkaCdiEvents();
```

Comment:
Should we reset the counter at the end of each test?

Reply:
For this one.. it's saying "I don't care" .. so I didn't bother resetting the counter on this one.

Reply:
ok